### PR TITLE
Add styles

### PR DESCRIPTION
**Pull request by y2kbadbug**
Added UIKit, updated some templates.  themed the site a bit.  commit comment has full details.

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -30,10 +32,12 @@ def create_superuser(self, email, password, **kwargs):
 
 class User(AbstractBaseUser, PermissionsMixin):
     USERNAME_FIELD = 'email'
-
-    email = models.EmailField(unique=True)
+    display_name = models.TextField(unique=True, null=True, max_length=30)
+    email = models.EmailField(unique=True, max_length=75)
```
Comment (funkybob) [inline accounts/models.py:36]: Why not leave this at the default max of 254?
A commend from the django source:

```
# max_length=254 to be compliant with RFCs 3696 and 5321
```

File: accounts/forms.py [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
 
 
 class RegistrationForm(forms.ModelForm):
-
     class Meta:
         model = User
-        fields = ['email', ]
+        fields = ['display_name', 'full_name', 'email']
+        widgets = {
+            'email': forms.TextInput(attrs={'class': 'uk-form-width-medium', 'required': 'required'}),
```
Comment (funkybob) [inline accounts/forms.py:11]: I'd really prefer we used django-sniplates instead of this tedium in forms.

File: accounts/views.py [hunk 3]
```diff
@@ -11,7 +15,23 @@ class RegistrationView(CreateView):
     template_name = 'registration/register.html'
 
     def form_valid(self, form):
+        form = self.form_class(data=self.request.POST)
```
Comment (funkybob) [inline accounts/views.py:18]: Why are you recreating the form instance, when a validated one is already passed to this method?

**Comment by funkybob** [comment]
This PR does _far_ more than the description suggests.
So far I've seen it updates the User model, adds a Settings model, rewrites the migrations, adds a flag to control if registration requires email verification, and vendors a copy of Codemirror...

File: accounts/models.py (hunk 1 above)
Comment (y2kbadbug) [inline accounts/models.py:36]: For usernames you don't want anything over 30.   I believe the email one was already set to 75, but don't care if it is 254 or what not.

File: accounts/forms.py (hunk 2 above)
Comment (y2kbadbug) [inline accounts/forms.py:11]: You want to promote your own stuff, that's fine.  I have enough on my plate already to be having to dig deeper into something else.  I did it the way I did because it was easy and I was able to make it look exactly how I wanted.  I won't theme anymore if that's the case and can revert the changes if you wish.

File: accounts/views.py (hunk 3 above)
Comment (y2kbadbug) [inline accounts/views.py:18]: That was a mistype, I'll fix tomorrow

File: accounts/forms.py (hunk 2 above)
Comment (funkybob) [inline accounts/forms.py:11]: Don't sweat it... I can make widget files from the templates you've made already...
